server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/mailbox/MailboxNamespaceTest.java

```diff
@@ -41,19 +43,19 @@ public void delegatedShouldThrowWhenNullOwner() throws Exception {
 
     @Test
     public void delegatedShouldThrowWhenEmptyOwner() throws Exception {
-        assertThatThrownBy(() -> MailboxNamespace.delegated(""))
+        assertThatThrownBy(() -> MailboxNamespace.delegated(Username.of("")))
             .isInstanceOf(IllegalArgumentException.class);
     }
 
     @Test
     public void delegatedShouldThrowWhenBlankOwner() throws Exception {
-        assertThatThrownBy(() -> MailboxNamespace.delegated("  "))
+        assertThatThrownBy(() -> MailboxNamespace.delegated(Username.of("  ")))
             .isInstanceOf(IllegalArgumentException.class);
     }
```

Comment:
I think they are not relevant anymore since only the `Username` builder is tested

Reply:
this test was failing, I fixed it. Not sure if we must enforce this in Username class, opinions?

Reply:
I think " " is not a valid user name, consequently, we should not be able to build it